stable/minecraft/ci/test-values.yaml

```diff
@@ -0,0 +1,3 @@
+# This must be overridden, since we can't accept this for the user.
+  eula: "TRUE"
```

Comment:
This must be `minecraftServer.eula`.

Reply:
I made this change, sorry it took over a month.